src/Components/FilterSection/CitizenshipPopover.tsx

```diff
@@ -39,49 +41,126 @@ const CitizenshipPopover = ({
     };
+
+    if (citizenshipType === 'green_card') {
+      // if the citizenshipType is `green_card`, then set green_card and all the gc_options to true or false
+      // i.e. green_card and all the gc_options should be the same when citizenshipType is `green_card`
+      updatedCitizenshipFilterIsChecked = {
+        ...updatedCitizenshipFilterIsChecked,
+        gc_5plus: !isChecked,
+        gc_18plus_no5: !isChecked,
+        gc_under18_no5: !isChecked,
+        gc_under19_pregnant_no5: !isChecked,
+      };
+    }
     const typedUpdatedCitizenshipFilterIsChecked = Object.keys(updatedCitizenshipFilterIsChecked) as CitizenLabels[];
     const selectedCitizenshipFilters = typedUpdatedCitizenshipFilterIsChecked.filter((citizenshipType) => {
       return updatedCitizenshipFilterIsChecked[citizenshipType];
     });
 
-    updateFilter({
-      name: 'citizen',
-      filter: {
-        id: 1,
-        columnField: 'citizenship',
-        operatorValue: 'customCitizenshipOperator',
-        value: selectedCitizenshipFilters,
-      },
-    });
+    //update the MUI filter that is being passed to the citizenship column
+    if (hasAtLeastOneCitizenshipFilter(updatedCitizenshipFilterIsChecked)) {
+      updateFilter({
+        name: 'citizen',
+        filter: {
+          id: 1,
+          columnField: 'citizenship',
+          operatorValue: 'customCitizenshipOperator',
+          value: selectedCitizenshipFilters,
+        },
+      });
+    } else {
+      // set the citizenship filter back to the default
+      updateFilter({
+        name: 'citizen',
+        filter: {
+          id: 1,
+          columnField: 'citizenship',
+          operatorValue: 'customCitizenshipOperator',
+          value: ['citizen'],
+        },
+      });
+    }
 
+    //update citizenshipFilterIsChecked state
     setCitizenshipFilterIsChecked(updatedCitizenshipFilterIsChecked);
   };
 
   const typedCitizenshipFilterIsChecked = Object.keys(citizenshipFilterIsChecked) as CitizenLabels[];
-  const citizenshipCheckboxFilters = typedCitizenshipFilterIsChecked.map((citizenshipType) => {
-    return (
-      <FormControlLabel
-        key={citizenshipType}
-        className="popover"
-        label={citizenshipFilterFormControlLabels[citizenshipType]}
-        control={
-          <Checkbox
-            checked={citizenshipFilterIsChecked[citizenshipType]}
-            onChange={() => handleFilterSelect(citizenshipType)}
-          />
-        }
-      />
-    );
-  });
-
-  return <Stack>{citizenshipCheckboxFilters}</Stack>;
+
+  const renderMainAndSubFilters = (citizenshipFilters: Record<CitizenLabels, boolean>) => {
+    return typedCitizenshipFilterIsChecked.map((citizenshipType) => {
+      //here we need to add an sx prop to indent them if they're the gc_filters
+      const isGreenCardSubCitizenshipType = [
+        'gc_5plus',
+        'gc_18plus_no5',
+        'gc_under18_no5',
+        'gc_under19_pregnant_no5',
+      ].includes(citizenshipType);
+
```

Comment:
```suggestion
if (citizenshipFilters.greenCard && isGreenCardSubCitizenshipType) {
  return <></>;
}
```
Then you can remove the renderMainFilters function.